modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,67 @@
+.Prerequisites
+
+* Define the IntelliJ-based IDE container image to use: __<IntelliJ-container-image>__. It can be:
+** A prebuilt IntelliJ-based IDEs image:
++
+`quay.io/che-incubator/che-idea:latest`:: IntelliJ IDEA Community Edition
++
+`quay.io/che-incubator/che-pycharm:latest`:: PyCharm Community Edition
+** A custom editor image, see xref:building-intellij-editor-images.adoc[].
+
+.Procedure
+
+. In the root directory of the workspace, create a `+meta.yaml+` file with the following content:
```

Comment:
This step is problemetic:

* It is quite complex. 
* It will need maintenance, as the meta files are changing.

Would it be an option to document how to use the meta files present there instead:   https://github.com/che-incubator/jetbrains-editor-images/tree/meta

Reply:
Refactored using  https://github.com/che-incubator/jetbrains-editor-images/tree/meta in the lates commit. Is that OK?
I still can't test because of quotas: 

```
Unrecoverable event occurred: 'FailedCreate', 'Error creating: pods "workspaceeuj9zsz1y9ukwzxf.che-docs-86b5c7b66f-w8lwt" is forbidden: exceeded quota: for-ffloreth-compute, requested: limits.memory=9402089472, used: limits.memory=0, limited: limits.memory=7Gi', 'workspaceeuj9zsz1y9ukwzxf.che-docs-86b5c7b66f'
```